include/lsst/jointcal/AstromFit.h

```diff
@@ -100,83 +120,107 @@ public :
     //! Evaluates the chI^2 derivatives (Jacobian and gradient) for the current whatToFit setting.
     /*! The Jacobian is provided as triplets, the gradient as a dense
         vector. The parameters which vary are to be set using
-        AssignIndices.  */
+        assignIndices.  */
     void LSDerivatives(TripletList &tList, Eigen::VectorXd &rhs) const;
 
-    //! Set parameter groups fixed or variable and assign indices to each parameter in the big matrix (which will be used by OffsetParams(...).
+    /**
+     * Set parameter groups fixed or variable and assign indices to each
+     * parameter in the big matrix (which will be used by OffsetParams(...).
+     *
+     * @param[in]  whatToFit  Valid strings: "Distortions", "Positions",
+     *                        "Refrac", "PM" which define which parameter set is
+     *                        going to be variable when computing derivatives
+     *                        (LSDerivatives) and minimizing (minimize()).
+     */
     void assignIndices(const std::string &whatToFit);
 
-    //!The transformations used to propagate errors are freezed to the current state.
-    /*! The routine can be called when the mappings are roughly in place.
-      After the call, the transformations used to propage errors are no longer
-      affected when updating the mappings. This allows to have an exactly linear
-      fit, which can be useful. */
+    /**
+     * The transformations used to propagate errors are freezed to the current
+     * state. The routine can be called when the mappings are roughly in place.
+     * After the call, the transformations used to propage errors are no longer
+     * affected when updating the mappings. This allows to have an exactly linear
+     * fit, which can be useful.
+     */
     void freezeErrorScales() {_distortionModel->freezeErrorScales();}
 
-    //! Offsest the parameters by the requested quantities. The used parameter layout is the one from the last call to AssignIndices or Minimize().
-    /*! There is no easy way to check that the current setting of
-        whatToFit and the provided Delta vector are compatible. We can
-        only test the size. */
+    /**
+     * Offsest the parameters by the requested quantities. The used parameter
```

Comment:
Fixed typo (and in photomFit.h). I'll leave the voicing as is (converting it from Frenglish was the first step), but I'll try to remember for the future.